qutebrowser/misc/sessions.py

```diff
@@ -459,6 +484,42 @@ def list_sessions(self):
                 sessions.append(base)
         return sorted(sessions)
 
+    def add_url_to_session(self, name):
+        """Appends the current tab's URL to a named session."""
+        main_window = objreg.get(
+            'main-window',
+            scope='window',
+            window='current'
+        )
+        tabbed_browser = objreg.get(
+            'tabbed-browser',
+            scope='window',
+            window='current'
+        )
+
+        try:
+            data = self._load_session_file(name)
+        except SessionNotFoundError:
+            data = {'windows': [
+                self._get_window_properties(
+                    main_window,
+                    tabbed_browser.private,
+                    True
+                )
```

Comment:
this is a nitpick, but there's a lot of empty space here, you should be able to slurp it into `True)]}`